src/CMakeLists.txt

```
@@ -31,7 +31,7 @@ set(SRC
     stdlib_experimental_ascii.f90
     stdlib_experimental_error.f90
     stdlib_experimental_kinds.f90
-    stdlib_experimental_system.F90
+    stdlib_experimental_system.F90 # Captital extension for a reason?
```

```suggestion
    stdlib_experimental_system.F90
```

The extension `.F90` triggers the C preprocessor before compilation